index.html

```diff
@@ -28,8 +28,10 @@ <h1>Hello Microverse Project</h1>
             </ul>
         </nav>
 
-        <div class="main_body flex-box body-text">
+        <div class="main_body flex-box flex-dir-column body-text">
             <p>Hello to all in Microverse!</p>
```

Comment:
- [ ] Kindly consider changing this p into an `h1 ` as requested in the project requirements to fulfill them

Reply:
Ok, that makes all sense to make the use of html tags more semantical